setup.cfg

```
@@ -74,9 +74,10 @@ envlist =
     mypy
     # tests against released versions
     py{36,37,38,39,310}-dj{22,31,32,40,41}-redislatest
+    py{311}-dj{41}-redislatest
     # tests against unreleased versions
-    py310-dj40-redismaster
-    py310-djmain-redis{latest,master}
+    py311-dj40-redismaster
```

I think you wanted to test against `django 4.1`

```suggestion
    py311-dj41-redismaster
```